VideoLocker/res/layout/edx_spinner_item.xml

```diff
@@ -12,4 +12,5 @@
     android:singleLine="true"
     android:textColor="@color/edx_grayscale_neutral_dark"
     android:textSize="@dimen/edx_x_small"
+    android:textDirection="locale"
```

Comment:
Oh, and in this case it's actually a layout issue; Spinners have an arrow on one side and text on the other. Without setting this to locale, the text is aligned beside the arrow, rather than opposite it.